py/kubeflow/kubeflow/cd/jwa.py

```diff
@@ -0,0 +1,48 @@
+""""Argo Workflow for building Jupyter web app's OCI image using Kaniko"""
+import os
+
+from kubeflow.kubeflow import ci
+from kubeflow.testing import argo_build_util
+
+TAG = os.getenv("PULL_BASE_SHA", "kaniko-local-test")
+AWS_REGISTRY = "public.ecr.aws/j1r0q0g6/jupyter-web-app"
```

Comment:
NIT: since WG-notebooks have multiple ECR registries (7 now, maybe more in the future), I would recommend you maintain a static configmap or mapping in other python files.

E.g, py/kubeflow/kubeflow/cd/config.py

Reply:
That's a good idea, it will help us have all the names organized in one place.
Adding it

Reply:
@PatrickXYS I added another commit that introduces this `config.py` file.
PTAL and if you are OK I can resolve this conversation

Reply:
LGTM feel free to resolve